content/posts/2017/2017-10-12-tibbles.Rmd

```diff
@@ -0,0 +1,227 @@
+---
+title: "The Trouble with Tibbles"
+author: "Theo Roe"
+date: 2017-10-13
+output: html_document
+image: "img/2017/tibbles.png"
+draft: false
+---
+
+```{r setup,include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+knitr::opts_chunk$set(collapse = TRUE)
+```
+
+Let's get something straight, there isn't really any trouble with tibbles. I'm hoping you've noticed this is a play on 1967 Star Trek episode, "The Trouble with Tribbles". I've recently got myself a job as a Data Scientist, here, at [Jumping Rivers](https://www.jumpingrivers.com/). Having never come across tibbles until this point, I now find myself using them in nearly every R script I compose. Be that your timeless standard R script, your friendly Shiny app or an analytical Markdown document. 
+
+## What are tibbles?
+
+Presumably this is why you came here, right?
+
+Tibbles are a modern take on data frames, but crucially they are _still_ data frames. Well, what's the difference then? There's a quote I found somewhere on the internet that decribes the difference quite well;
+
+ _"keeping what time has proven to be effective, and throwing out what is not"_.   
+
+Basically, some clever people took the classic `data.frame`, shook it til the ineffective parts fell out, then added some new, more appropriate features. 
```

Comment:
In our course notes, when it's a function, add brackets. e.g `data.frame()`. When referring to a package, use bold, __tibble__